pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_outlook_ios/guide.pt-pt.md

```diff
@@ -0,0 +1,169 @@
+---
+title: "E-mail Pro - Configurar a sua conta de e-mail no Outlook para iOS"
+excerpt: "Saiba como configurar o seu endereço E-mail Pro na aplicação móvel Outlook para iOS"
+updated: 2025-02-13
+---
+
+<style>
+.w-400 {
+  max-width:400px !important;
+}
+.h-600 {
+  max-height:600px !important;
+}
+</style>
+
+## Objetivo
+
+As contas E-mail Pro podem ser configuradas em vários softwares de e-mail compatíveis. Isto permite-lhe utilizar o seu endereço de e-mail a partir do dispositivo que preferir. A aplicação Outlook da Microsoft para iOS está disponível gratuitamente a partir da App Store da Apple.
+
+**Saiba como configurar o seu endereço de E-mail Pro na aplicação móvel Outlook para iOS**
+
+> [!warning]
+>
+> A OVHcloud oferece-lhe serviços cuja configuração, gestão e responsabilidade é da sua responsabilidade. Assim, deverá assegurar o seu bom funcionamento.
+>
+> Este guia fornece as instruções necessárias para realizar as operações mais habituais. No entanto, se encontrar dificuldades, recomendamos que recorra a um [parceiro especializado](https://marketplace.ovhcloud.com/c/support-collaboration) e/ou que contacte o editor do serviço. Não poderemos proporcionar-lhe assistência técnica. Para mais informações, consulte [Vá mais longe](#gofurther) deste guia.
```

Comment:
```suggestion
> Este guia fornece as instruções necessárias para realizar as operações mais habituais. No entanto, se encontrar dificuldades, recomendamos que recorra a um [parceiro especializado](https://marketplace.ovhcloud.com/c/support-collaboration) e/ou que contacte o editor do serviço. Não poderemos proporcionar-lhe assistência técnica. Para mais informações, consulte [Quer saber mais?](#gofurther) deste guia.
```

Reply:
I think you copy/paste this paragraph. I suggest to you to do a search and replace for all of all email and collaborative solutions.
This just to avoid for the future proofs to do this commit suggestion.